spec/models/activity_spec.rb

```diff
@@ -9,35 +9,81 @@
-      it "should set the recipient_region to 998" do
-        activity = build_stubbed(:activity, hierarchy: fund)
-        activity.set_hierarchy_defaults
-        expect(activity.recipient_region).to eq("998")
-      end
-
-      it "should set the flow to 10" do
-        activity = build_stubbed(:activity, hierarchy: fund)
-        activity.set_hierarchy_defaults
-        expect(activity.flow).to eq("10")
-      end
-
-      it "should set the tied_status to 5" do
-        activity = build_stubbed(:activity, hierarchy: fund)
-        activity.set_hierarchy_defaults
-        expect(activity.tied_status).to eq("5")
-      end
+  describe "validations" do
+    context "when title is blank" do
+      subject { build(:activity, title: nil, wizard_status: :purpose) }
+      it { should validate_presence_of(:title) }
+    end
+
+    context "when description is blank" do
+      subject { build(:activity, description: nil, wizard_status: :purpose) }
+      it { should validate_presence_of(:description) }
+    end
+
+    context "when sector is blank" do
+      subject { build(:activity, sector: nil, wizard_status: :sector) }
+      it { should validate_presence_of(:sector) }
+    end
+
+    context "when status is blank" do
+      subject { build(:activity, status: nil, wizard_status: :status) }
+      it { should validate_presence_of(:status) }
+    end
+
+    context "when planned_start_date is blank" do
+      subject { build(:activity, planned_start_date: nil, wizard_status: :dates)}
+      it { should_not validate_presence_of(:planned_start_date) }
+    end
+
+    context "when planned_end_date is blank" do
+      subject { build(:activity, planned_end_date: nil, wizard_status: :dates)}
+      it { should_not validate_presence_of(:planned_end_date) }
+    end
+
+    context "when actual_start_date is blank" do
+      subject { build(:activity, actual_start_date: nil, wizard_status: :dates)}
+      it { should_not validate_presence_of(:actual_start_date) }
+    end
+
+    context "when actual_end_date is blank" do
+      subject { build(:activity, actual_end_date: nil, wizard_status: :dates)}
+      it { should_not validate_presence_of(:actual_end_date) }
+    end
+
+    context "when recipient_region is blank" do
+      subject { build(:activity, recipient_region: nil, wizard_status: :country) }
+      it { should validate_presence_of(:recipient_region) }
+    end
+
+    context "when flow is blank" do
+      subject { build(:activity, flow: nil, wizard_status: :flow) }
+      it { should validate_presence_of(:flow) }
+    end
+
+    context "when finance is blank" do
+      subject { build(:activity, finance: nil, wizard_status: :finance) }
+      it { should validate_presence_of(:finance) }
+    end
+
+    context "when tied_status is blank" do
+      subject { build(:activity, tied_status: nil, wizard_status: :tied_status) }
+      it { should validate_presence_of(:tied_status) }
+    end
+
+    context "when the wizard_status is complete" do
```

Comment:
As I explained on this commit it's premature and not strictly needed. I felt more confident having test coverage in place from the off to ensure we have a mechanism for calling all validations at once. This is in the Wicked docs and will be helpful in any backfilling we have to do of old activities.